src/nvim/path.c

```diff
@@ -1349,11 +1349,11 @@ static int expand_backtick(
   // Create the command: lop off the backticks.
   char_u *cmd = vim_strnsave(pat + 1, STRLEN(pat) - 2);
 
-  if (*cmd == '=')          /* `={expr}`: Expand expression */
-    buffer = eval_to_string(cmd + 1, &p, TRUE);
-  else
-    buffer = get_cmd_output(cmd, NULL,
-        (flags & EW_SILENT) ? kShellOptSilent : 0, NULL);
+  if (*cmd == '=') {          // `={expr}`: Expand expression
+    buffer = eval_to_string(cmd + 1, &p, true);
+  } else {
+    buffer = get_cmd_output(cmd, NULL, (flags & EW_SILENT) ? kShellOptSilent : 0, NULL);
```

Comment:
It was on a separate line to not exceed 80 characters.
Did that style rule change?

Reply:
Oh yeah, that was my fault :). Upped it to 100 characters.